doc/Makefile

```
@@ -15,14 +15,44 @@
-SPHINXBUILD  ?= sphinx-build
-SOURCEDIR    = .
-BUILDDIR     = _build
+SOURCEDIR     = .
+BUILDDIR      = _build
+BUILDDIR_CMS  = _build-cms
+
+NODE_MODULES  = $(BUILDDIR_CMS)/node_modules
+MMDC          = $(NODE_MODULES)/.bin/mmdc
+PUPPETEER_CFG = $(BUILDDIR_CMS)/puppeteer-config.json
+
+.PHONY: help html cms-html install-mermaid clean
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)